### PR TITLE
Add slack notifier

### PR DESCRIPTION
**Pull request by ztanis**
- add slack notifier
- readme updates

### CODE REVIEW

File: notices/slacknotice/slack_test.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+	"log"
+	"os"
+	"strconv"
+	"time"
+	"net/http/httptest"
+	"net/http"
+	"fmt"
+	"io/ioutil"
+	"github.com/stretchr/testify/assert"
+	"github.com/pubnative/statusboard/notices"
```
Comment (kostyantyn) [inline notices/slacknotice/slack_test.go]: please, split list of included packages by 3 groups: builtin, external, internal

File: notices/slacknotice/slack_test.go [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(200)
+		body, _ := ioutil.ReadAll(r.Body)
+		messages <- string(body)
+		fmt.Fprintln(w, `ok`)
+	}))
+	return messages, ts
+}
```
Comment (kostyantyn) [inline notices/slacknotice/slack_test.go]: please, add new line to ignore this git error

**Comment by kostyantyn** [comment]
please, run `go fmt ./...` to format code

**Comment by ztanis** [comment]
thx, reformated

**Comment by kostyantyn** [comment]
:+1:

**Pull request merged**